### PR TITLE
Refactor `/globals/app-nav.js` test for jsonapi

### PR DESCRIPTION
**Pull request by nmajor25**

### CODE REVIEW

File: test/integration/globals/app-nav.js [hunk 1]
```diff
@@ -14,9 +19,15 @@ context('App Nav', function() {
   specify('display non-manager nav', function() {
     cy
       .routeCurrentClinician(fx => {
-        fx.data.id = '123456';
-        fx.data.attributes.enabled = true;
-        fx.data.relationships.role = { data: { id: '33333' } };
+        fx.data = getCurrentClinician({
+          attributes: {
+            enabled: true,
```
Comment (paulfalgout) [inline test/integration/globals/app-nav.js]: this enable isn't necessary. all the test specific clinicians are enabled.

File: test/integration/globals/app-nav.js [hunk 2]
```diff
@@ -528,13 +539,13 @@ context('App Nav', function() {
         last_active_at: testTs(),
       },
       relationships: {
-        team: { data: { id: '11111' } },
+        team: getRelationship(teamCoordinator),
         workspaces: { data: _.times(10, n => {
```
Comment (nmajor25) [inline test/integration/globals/app-nav.js]: Not sure how to handle this one. I'm having a hard time getting the data formatted correctly using `getWorkspace()`/`getWorkspaces()` 🤔

File: test/integration/globals/app-nav.js [hunk 3]
```diff
@@ -965,11 +975,11 @@ context('App Nav', function() {
         last_active_at: testTs(),
       },
       relationships: {
-        team: { data: { id: '11111' } },
+        team: getRelationship(teamCoordinator),
         workspaces: { data: [{ id: workspaceOne.id }] },
```
Comment (nmajor25) [inline test/integration/globals/app-nav.js]: Same comment as above.

Comment (paulfalgout) [reply test/integration/globals/app-nav.js]: pretty sure this is just `getRelationship([workSpaceOne])`

File: test/integration/globals/app-nav.js (hunk 2 above)
Comment (paulfalgout) [reply test/integration/globals/app-nav.js]: probably just `getRelationships(_.times(10.... return { id: uuid(...) }..), 'workspaces'),`

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: test/integration/globals/app-nav.js (hunk 1 above)
Comment (nmajor25) [reply test/integration/globals/app-nav.js]: That's been removed 👍

File: test/integration/globals/app-nav.js (hunk 3 above)
Comment (nmajor25) [reply test/integration/globals/app-nav.js]: Yup, that works 👍

File: test/integration/globals/app-nav.js (hunk 2 above)
Comment (nmajor25) [reply test/integration/globals/app-nav.js]: Yup, that works 👍

**Pull request merged**